components/d2l-applied-filters/d2l-applied-filters.js

```diff
@@ -237,6 +261,8 @@ class D2lAppliedFilters extends RtlMixin(LocalizeStaticMixin(LitElement)) {
 			<div class="d2l-applied-filters-wrapper">
 				<span id="d2l-applied-filters-label" class="d2l-applied-filters-applied-filters-label d2l-body-compact">${this.localize('appliedFilters')}</span>
 				${filters}
+				<!-- <d2l-button-subtle text="${this.localize('clearFilters')}" ?hidden="${this._hideClearFiltersButton}" @click="${this._clearFiltersClicked}"></d2l-button-subtle> -->
```

Comment:
Just a reminder so that you don't forget to remove commented out code before finalizing this

Reply:
Good catch, I added that just now by accident. I removed it